src/BlazorWebView/src/Maui/Windows/BlazorWebViewHandler.Windows.cs

```diff
@@ -42,16 +42,14 @@ private void StartWebViewCoreIfPossible()
 				throw new InvalidOperationException($"Can't start {nameof(BlazorWebView)} without native web view instance.");
 			}
 
-			var assetConfig = Services!.GetRequiredService<BlazorAssetsAssemblyConfiguration>()!;
-
 			// We assume the host page is always in the root of the content directory, because it's
 			// unclear there's any other use case. We can add more options later if so.
 			var contentRootDir = Path.GetDirectoryName(HostPage!) ?? string.Empty;
 			var hostPageRelativePath = Path.GetRelativePath(contentRootDir, HostPage!);
 
-			var fileProvider = new ManifestEmbeddedFileProvider(assetConfig.AssetsAssembly, root: contentRootDir);
+			var mauiAssetFileProvider = new NullFileProvider();
```

Comment:
Just confirming that the `NullFileProvider` is corrent.

Reply:
Yes it is correct. I will add comments. In Windows we don't currently use the file provider at all because that's sync-only, whereas reading files in WinUI is async-only. Unfortunately I couldn't get _true_ async to work just yet, but the code is all in the right place.